turbo.json

```diff
@@ -8,6 +8,10 @@
 			"dependsOn": ["^build"],
 			"outputs": ["dist/**"]
 		},
+		"build:backend": {},
+		"build:frontend": {},
+		"test:backend": {},
+		"test:frontend": {},
```

Comment:
I don't think this alone is enough. we need to define what workspaces to run the `build` command in. We could try updating `https://github.com/n8n-io/n8n/blob/master/package.json#L28` to use `turbo run --filter=` instead of `pnpm --filter=`, but unlike pnpm, turbo will still run the tests for editor, because `n8n` depends on that package.